src/common.js

```diff
@@ -30,8 +31,10 @@ var state = {
 };
 exports.state = state;
 
+// XXX: This needs a new home
 delete process.env.OLDPWD; // initially, there's no previous directory
```

Comment:
It needs to be run only once (clears out state). Doesn't matter where that happens. I think it's not a big issue, unless this file can be included multiple times for one shelljs instance